src/containers/Bookings.js

```diff
@@ -2,22 +2,37 @@
 import React, { Component } from "react";
 import Search from "../components/search/Search.js";
 import Results from "../components/results/Results.js";
-import * as FakeBookings from "../data/fakeBookings.json";
+
+// import * as FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      FakeBookings: '',
-      term: '',
+      FakeBookings: [],
+      term: null,
       result: false,
       direction: {
-        sortin: 'asc'
+        sortin: 'asc',
+        error: null,
       },
-      text: ''
+      text: null,
     }
     this.searchHandler = this.searchHandler.bind(this)
     this.sortBy = this.sortBy.bind(this)
+    this.sortByNumber = this.sortByNumber.bind(this)
+  }
+
+
+  componentDidMount() {
+    fetch(`http://192.168.43.95:8080/api/Customersjoinreservation`)
+      .then((res) => res.json())
+      .then((FakeBookings) => { this.setState({ FakeBookings: FakeBookings.customers }) })
```

Comment:
The bookings aren't fake now 🙂. So I would rename this to `bookings` or maybe `data`.